contracts/PegStability/PegStability.sol

```diff
@@ -41,14 +45,14 @@ contract PegStability is AccessControlledV8, ReentrancyGuardUpgradeable {
         IN,
         OUT
     }
+    uint256 public constant BASIS_POINTS_DIVISOR = 10000; // fee is in basis points
+    uint256 public constant MANTISSA_ONE = 1e18;
+    uint256 public immutable ONE_DOLLAR; // Our oracle is returning amount depending on the number of decimals of the stable token. (36 - asset_decimals). E.g. 8 decimal asset = 1e28
```

Comment:
I am not sure if immutable variables should be with capital. And if it has to, we should review the rest of immutable variables in this file